BuildBom/SolutionGenerator.cs

```diff
@@ -77,6 +84,22 @@ public override bool Execute()
                         bool projectResult = base.Execute();
                         PackagesConfigPath = ""; // reset to use the project packages file.
                         result = result && projectResult;
+
+                        if (projectResult && GenerateMergedBdio)
+                        {
+                            string bdioFilePath = $"{OutputDirectory}{Path.DirectorySeparatorChar}{HubProjectName}.jsonld";
```

Comment:
Use Path.combine. https://msdn.microsoft.com/en-us/library/fyy7a5kt(v=vs.110).aspx

It handles special cases that concatenation does not.